src/pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -134,16 +123,16 @@ def get_coupled_variables(self, variables):
         elif SEI_option == "electron-migration limited":
             # Scott Marquis thesis (eq. 5.94)
             eta_inner = delta_phi - phase_param.U_inner
```

Comment:
I noticed `U_outer` doesn't get used anywhere. Deleting `U_inner` and `U_outer` and just having `U_sei` would reduce the total number of parameters, but I'm aware that `U_inner` and `U_sei` are for different mechanisms

Reply:
Yes, I spotted that too. I guess the main question is, if we switch to just `U_sei`, which parameter value should we use? I believe at the moment one is set at 0.1 V and the other at 0.8 V.